setup.py

```diff
@@ -39,6 +39,7 @@
         'django-bootstrap-datepicker-plus',
         'django-crispy-forms',
         'django-extensions',
+        'django-import-export',
```

Comment:
It is important to update the `requirements.lock` in `ok_tools.deployment` afterwards too.